package.json

```diff
@@ -37,7 +37,9 @@
     "test.pw.headless-chrome-108": "nx e2e-chrome-108 headless",
     "test.pw.headless.ci": "nx e2e headless",
     "test.headless.ci": "nx component-test-ci headless",
-    "test.utils": "nx test utils"
+    "test.utils": "nx test utils",
+    "test:qwik-ci": "nx run-many --target=test --exclude=cli && nx run-many --target=e2e",
+    "build:qwik-ci": "nx run-many --target=build -p packages/*"
```

Comment:
pay attention that the convention in Qwik and Qwik UI is `build.qwik-ci` . so "dot" instead of ":" 

plus, let's sort it alphabetically so let's move the "build" to the top  in the right place

Thanks buddy!

Reply:
@shairez thanks done!